lib/dart_cid.dart

```diff
@@ -1,3 +1,4 @@
 library dart_cid;
 
-export 'src/cid.dart';
\ No newline at end of file
+export 'src/cid.dart';
+export 'src/decode_cid.dart';
```

Comment:
Care to elaborate! The two needed functions are inside the `CID` static class in `cid.dart`.
No need to export the functions in `decode_cid.dart`, they are meant to be private to the library 🤔

Reply:
Doesn't it cid info object come from decode cod file?

Reply:
It does but it's invoked by the static `CID` class that is made "public".

https://github.com/dwyl/dart_cid/blob/2a68ddcb16c1b44d9e757f2fcf789e369644eca0/lib/src/cid.dart#L23-L25

So there's no need to expose `decode_cid.dart` to the public, we already do that with the static `CID` class 😄 .

And that's awesome news, super happy to hear that! 🥳